src/pages/experiments/[experimentId]/data-exploration/components/heatmap/HeatmapPlot.jsx

```diff
@@ -16,6 +16,7 @@ import { loadCellSets } from '../../../../../../redux/actions/cellSets';
 import { loadComponentConfig } from '../../../../../../redux/actions/componentConfig';
 
 import { union } from '../../../../../../utils/cellSetOperations';
+import SetOperations from '../../../../../../utils/setOperations';
 
 const COMPONENT_TYPE = 'InteractiveHeatmap';
```

Comment:
I suggest that we directly use `COMPONENT_TYPE` instead of hardcoding `interactiveHeatmap` in all dispatch actions, unless there is some specific reason why we are doing it this way that I am not seeing.

Reply:
In order to use this and keep the naming consistent I changed COMPONENT_TYPE from 'InteractiveHeatmap' to 'interactiveHeatmap', as far as I could see it is ok, but if you believe this could cause any issues let me know.

Reply:
provided that the group by and metadata settings work when used in the heatmap options, everything should be fine.